js/main.js

```diff
@@ -21,3 +21,78 @@ formSelector.addEventListener('submit', function (event) {
   formSelector.reset();
 
 });
+
+function addEntry(entry) {
+
+  var divOne = document.createElement('div');
+  divOne.setAttribute('class', 'dummy-row');
+  var divImage = document.createElement('div');
+  divImage.setAttribute('class', 'dummy-image');
+  divOne.appendChild(divImage);
+  var listOne = document.createElement('li');
+  divImage.appendChild(listOne);
+  var photo = document.createElement('img');
+  photo.setAttribute('src', entry.url);
+  photo.setAttribute('class', 'd-photo');
+  photo.setAttribute('alt', 'placeholder');
+  listOne.appendChild(photo);
+  var divText = document.createElement('div');
+  divText.setAttribute('class', 'dummy-text');
+  divOne.appendChild(divText);
+  var listTwo = document.createElement('li');
+  divText.appendChild(listTwo);
+  var heading = document.createElement('h2');
+  heading.textContent = entry.title;
+  listTwo.appendChild(heading);
+  var listThree = document.createElement('li');
+  listThree.textContent = entry.textbox;
+  divText.appendChild(listThree);
+
+  return divOne;
+}
+
+var dummySelector = document.querySelector('.dummy');
+
+window.addEventListener('DOMContentLoaded', function (event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    dummySelector.appendChild(addEntry(data.entries[i]));
+  }
+  restore();
+});
+
+var htwoEntries = document.querySelector('.hEntries');
+
+var displayList = document.querySelector('.dummy');
+
+var newButton = document.querySelector('.new');
+newButton.addEventListener('click', function (event) {
+  formSelector.className = 'main-form';
+  displayList.className = 'hidden';
+  data.view = 'entry-form';
+  htwoEntries.className = 'hidden';
+  newButton.className = 'hidden';
+  restore();
+});
+
+var views = document.querySelector('.a-entries');
+views.addEventListener('click', function (event) {
+  formSelector.className = 'hidden';
+  displayList.className = 'dummy';
+  data.view = 'entries';
+  htwoEntries.className = 'hEntries';
+  newButton.className = 'new';
+
+  restore();
+
+});
+
+function restore(event) {
```

Comment:
The name of this function doesn't describe its purpose. You should name this `changeView`